drivers/i2c/i2c_ll_stm32.c

```diff
@@ -233,6 +233,8 @@ static const struct i2c_driver_api api_funcs = {
 #endif
 };
 
+#ifdef CONFIG_PM_DEVICE
```

Comment:
Please move this directly to the commit where you introduce `i2c_stm32_suspend`

Reply:
done